examples/auction/src/lib.rs

```diff
@@ -3,206 +3,191 @@ use core::fmt::Debug;
 
 /// # Implementation of an auction smart contract
```

Comment:
The contract description should probably be a module level doc comment, which is done using 
```
//! 
```
for every line instead of `///`